doc/source/rllib/rllib-learner.rst

```diff
@@ -0,0 +1,459 @@
+.. include:: /_includes/rllib/announcement.rst
+
+.. include:: /_includes/rllib/we_are_hiring.rst
+
+.. |tensorflow| image:: images/tensorflow.png
+    :class: inline-figure
+    :width: 16
+
+.. |pytorch| image:: images/pytorch.png
+    :class: inline-figure
+    :width: 16
+
+
+Learner (Alpha)
+===============
+
+:py:class:`~ray.rllib.core.learner.learner.Learner` allows you to abstract the training 
+logic of RLModules. It supports both gradient-based and non-gradient-based updates (e.g.
+polyak averaging, etc.) The API enables you to distribute the Learner using data-
+distributed parallel (DDP). The Learner achieves the following:
+
+
+(1) Facilitates gradient-based updates on `RLModule`.
```

Comment:
Can we link to the RL Modules Guide here?

Reply:
done.